phpunit.xml

```diff
@@ -9,6 +9,7 @@
   >
   <php>
     <ini name="error_reporting" value="-1" />
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="62" /> <!-- there are 62 known issues that can't cause CI to fail -->
```

Comment:
I think you can remove this, and rather merge this PR into https://github.com/ezsystems/ezpublish-kernel/pull/1753 to keep track of it there.